### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by barnabasbartha**

### CODE REVIEW

File: src/main/client/bullet/default-bullet.ts [hunk 1]
```diff
@@ -61,25 +63,30 @@ export class Bullets extends Phaser.Physics.Arcade.Group {
...
             volume: 0.3,
             detune: -50,
          }),
-         scene.sound.add(Keys.BASIC_GUN_SOUND, {
+         scene.sound.add(key, {
             volume: 0.3,
             detune: 50,
          }),
-         scene.sound.add(Keys.BASIC_GUN_SOUND, {
-            volume: 0.3,
```
Comment (dolanmiu) [inline src/main/client/bullet/default-bullet.ts:80]: why is 100 detune removed

Comment (barnabasbartha) [reply src/main/client/bullet/default-bullet.ts:80]: There was no 100 detune. I kept the values

File: src/main/client/bullet/default-bullet.ts [hunk 2]
```diff
@@ -61,25 +63,30 @@ export class Bullets extends Phaser.Physics.Arcade.Group {
...
-            detune: 100,
```
Comment (dolanmiu) [inline src/main/client/bullet/default-bullet.ts:81]: says its deleted

Comment (barnabasbartha) [reply src/main/client/bullet/default-bullet.ts:81]: Weird, I cant see them in intellij. I'll fix this

Comment (barnabasbartha) [reply src/main/client/bullet/default-bullet.ts:81]: Fixed, thanks.

**Review by dolanmiu — APPROVED** [review]

**Pull request merged**